game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModel.java

```diff
@@ -148,26 +144,16 @@ public Object getValueAt(final int rowIndex, final int columnIndex) {
         return description.getPlayerCount();
       case P:
         return (description.getPassworded() ? "*" : "");
-      case B:
-        return description.isBot() ? "-" : "";
       case GV:
         return description.getGameVersion();
-      case EV:
-        return description.getEngineVersion();
       case Status:
         return description.getStatus();
       case Comments:
         return description.getComment();
-      case Started:
-        return formatBotStartTime(description.getStartDateTime());
       case GUID:
         return gameList.get(rowIndex).getFirst();
       default:
         throw new IllegalStateException("Unknown column:" + column);
     }
   }
-
-  private static String formatBotStartTime(final Instant instant) {
-    return TimeManager.getLocalizedTimeWithoutSeconds(LocalDateTime.ofInstant(instant, ZoneOffset.systemDefault()));
```

Comment:
Is this method still used elsewhere?

Reply:
Pretty sure not, would not compile if so.

Reply:
I meant the TileManager#getLocalizedTimeWithoutSeconds method

Reply:
You're right, this was the one usage. We are bringing back started column momentarily at least, I inlined this method to avoid the extra indirection.